versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java

```diff
@@ -58,7 +58,6 @@ public VersionCommon(VersionSupport versionSupport) {
             }
 
             // 1.12 drop listeners
-            //if (versionSupport.getVersion() == 5){
```

Comment:
why you doing that? that check is here for a reason

Reply:
That was accident my bad

Reply:
I went looking for something in that class I think I went to leave a comment but then I accidently commented it out